cypress/e2e/tests/migration/dynamic-report/issues/singleApplication/sort.test.ts

```diff
@@ -0,0 +1,49 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(["@tier2"], "Issues sort validations", function () {
+    let application: Analysis;
+    const sortByList = ["Issue", "Category", "Effort", "Affected files"];
+
+    before("Load data, create Analysis instance and run analysis", function () {
+        login();
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+    });
+
+    it("Source Analysis on daytrader application", function () {
+        application = new Analysis(
+            getRandomApplicationData("daytrader-app", {
+                sourceData: this.appData["daytrader-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["source+dep_analysis_on_daytrader-app"])
+        );
+        application.create();
```

Comment:
The application is not being deleted as part of the clean up .

Reply:
this is fixed already